_docs_use-cases/use-case-consumption-delete-a-peer-shared-attribute-and-notify-peer.md

```diff
@@ -52,7 +52,8 @@ This use-case allows you to delete a peer shared Attribute, i.e. a [LocalAttribu
 - The peer shared Attribute will be deleted.
 - All predecessors of the peer shared Attribute will be deleted.
 - If the peer shared Attribute was succeeded, the `succeeds` property of the successor will be set to undefined.
-- A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) is sent to the owner of the Attribute, informing them that you deleted their Attribute. Technically, the own shared Attribute of the owner and all predecessors will get a `deletionInfo` with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
+- A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) with [PeerSharedAttributeDeletedByPeerNotificationItem]({%link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem) is sent to the owner of the Attribute, informing them that you deleted their Attribute. Technically, the own shared Attribute of the owner and all predecessors will get a `deletionInfo` with `deletionStatus` `"DeletedByPeer"` and the time of receiving the Notification as `deletionDate`.
```

Comment:
I'd write: "A [Notification]({%link _docs_integrate/data-model-overview.md %}#notification) with **a** [PeerSharedAttributeDeletedByPeerNotificationItem]({%link _docs_integrate/data-model-overview.md %}#peersharedattributedeletedbypeernotificationitem)..."

Same for the "delete third party owned RelationshipAttribute" and "delete own shared Attribute" use cases.